ConcernsCaseWork/ConcernsCaseWork.Data/Gateways/ConcernsCaseGateway.cs

```diff
@@ -14,8 +14,17 @@ public ConcernsCaseGateway(ConcernsDbContext concernsDbContext)
 
         public ConcernsCase SaveConcernsCase(ConcernsCase concernsCase)
         {
-            _concernsDbContext.ConcernsCase.Update(concernsCase);
-            _concernsDbContext.SaveChanges();
+	        try
```

Comment:
QUESTION:

Do we need the try catch if we just re-throw the exception?

Reply:
debug code, removed